posthog-react-native/src/posthog-rn.ts

```diff
@@ -250,10 +251,10 @@ export class PostHog extends PostHogCore {
       if (OptionalReactNativeSessionReplay) {
         try {
           this._resetSessionId(OptionalReactNativeSessionReplay, sessionId)
-          this.logMsgIfDebug(() => console.info('PostHog Debug', `Session replay started with sessionId ${sessionId}.`))
+          this.logMsgIfDebug(() => console.info('PostHog Debug', `sessionId rotated from ${this._currentSessionId} to ${sessionId}.`))
```

Comment:
the session id rotation can happen even without session replay so the log should be session id generic